test/fixtures/dialects/oracle/bind_variables.yml

```diff
@@ -39,7 +39,7 @@ file:
             colon: ':'
             parameter: xyz
         - end_bracket: )
-- statement_terminator: ;
```

Comment:
The statement terminators be at the same nesting as the statement. There are a few places in this PR where you have the Delimiter as part of the statement, but the filesegment should handle statements and their terminators